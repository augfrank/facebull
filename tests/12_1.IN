M0   C0   C2  2363
M1   C0   C11  2649
M2   C1   C4  2747
M3   C2   C3  2832
M4   C3   C8  2427
M5   C3   C10  2884
M6   C3   C11  2882
M7   C4   C7  2865
M8   C4   C8  2960
M9   C5   C6  2674
M10   C6   C0  2886
M11   C6   C4  2977
M12   C6   C10  2994
M13   C7   C2  2083
M14   C7   C5  2211
M15   C7   C6  2866
M16   C8   C1  2543
M17   C8   C2  2105
M18   C8   C10  2748
M19   C9   C1  2012
M20   C9   C11  2996
M21   C10   C4  2385
M22   C10   C7  2974
M23   C10   C9  2381
M24   C11   C8  2843
