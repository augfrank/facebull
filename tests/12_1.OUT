31089
1 2 3 5 7 9 10 14 17 19 23 24
